pkg/network/setup/network_suite_test.go

```diff
@@ -51,3 +56,19 @@ func NewDomainWithBridgeInterface() *api.Domain {
 	}
 	return domain
 }
+
+type tempCacheCreator struct {
+	once   sync.Once
+	tmpDir string
+}
+
+func (c *tempCacheCreator) New(filePath string) *cache.Cache {
```

Comment:
I think it the third place I see this code copy pasted.
BTW, where the temp dir is removed?